### DISCUSSION TITLE
[Linux} Not showing in system tray

### DISCUSSION THREAD

**Discussion by oblongau** [ankurk91/google-chat-electron#112] [Q&A]
Linux Mint 20.3 MATE. When I close the chat window with File|Close To Tray (or Ctrl-W) nothing shows in the bottom right system tray (?) area. Should it? Am I missing something?

**Comment by ankurk91** [comment]
Please share more details.

App version
Installation method, snap?

**Reply by oblongau** [reply]
Sorry about missing information. Version 2.18.0. Installed today from [google-chat-electron_2.18.0_amd64.deb](https://github.com/ankurk91/google-chat-electron/releases/download/2.18.0/google-chat-electron_2.18.0_amd64.deb) using `sudo apt install ./google-chat-electron_2.18.0_amd64.deb` (from the downloaded directory).

Starts at boot with `/usr/lib/google-chat-electron/google-chat-electron --hidden`. Without running the program manually the System Monitor shows 6 mentions of google-chat-electron after boot. 

If I close all the processes (in case something is related to how it starts at boot) and then run the program followed by `File|Close To Tray` there is still no icon showing.

The program runs manually from a desktop launcher using `google-chat-electron`.

**Comment by ankurk91** [comment]
Thanks, i need to download the Mate ISO and install it in Virtual box to test it
